ui/app/constants/error-keys.js

```diff
@@ -1,3 +1,3 @@
 export const INSUFFICIENT_FUNDS_ERROR_KEY = 'insufficientFunds'
 export const GAS_LIMIT_TOO_LOW_ERROR_KEY = 'gasLimitTooLow'
-export const TRANSACTION_ERROR = 'transactionError'
+export const TRANSACTION_ERROR_KEY = 'transactionError'
```

Comment:
We should expand our ESLint rules to include something like `import/no-unresolved`